ui/components/app/add-network/index.scss

```
@@ -8,10 +8,13 @@
     }
   }
 
-  &__token-image {
-    margin-right: 7px;
-    height: 24px;
-    width: 24px;
+  &__warning-tooltip {
```

Adding waring icon css here
```suggestion
  &__warning-icon {
    color: var(--color-icon-muted);
  }

  &__warning-tooltip {
```

---

Done